src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.tsx

```diff
@@ -0,0 +1,94 @@
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { LocationOn } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactDetailsMailingTitle = styled(Typography)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingMainContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingIcon = styled(LocationOn)(({ theme }) => ({
+  margin: theme.spacing(1),
+  width: '14px',
+  height: '20px',
+  flexGrow: 1,
+}));
+
+const ContactDetailsMailingTexContainer = styled(Box)(({}) => ({
+  flexGrow: 8,
+}));
+
+const ContactDetailsMailingLabelTextContainer = styled(Box)(({}) => ({
+  display: 'flex',
+  marginTop: '10px',
+}));
+
+const ContactDetailsMailingLabel = styled(Typography)(({ theme }) => ({
+  color: theme.palette.text.hint,
+  marginRight: '5px',
+}));
+
+interface MailingProp {
+  data: ContactDetailsTabQuery;
+}
+
+export const ContactDetailsTabMailing: React.FC<MailingProp> = ({ data }) => {
+  const { t } = useTranslation();
+  const {
+    name,
+    primaryAddress: { street, city, state, postalCode },
+    greeting,
+    sendNewsletter,
+  } = data.contact;
+  return (
+    <Box>
+      <ContactDetailsMailingTitle variant="h6">
+        {t('Mailing')}
+      </ContactDetailsMailingTitle>
+      <ContactDetailsMailingMainContainer>
+        <ContactDetailsMailingIcon />
+        <ContactDetailsMailingTexContainer>
+          {/* Address Section */}
+          <Typography variant="subtitle1">{name}</Typography>
+          <Typography variant="subtitle1">{street}</Typography>
+          <Typography variant="subtitle1">
+            {`${city}, ${state} ${postalCode}`}
+          </Typography>
```

Comment:
So ```primaryAddress``` can be null, so we should probably have a check for that.
```suggestion
 {primaryAddress !== null ? (
 <Typography variant="subtitle1">{primaryAddress?.street}</Typography>
          <Typography variant="subtitle1">
            {`${primaryAddress?.city}, ${primaryAddress?.state} ${primaryAddress?.postalCode}`}
          </Typography>
 ) : null}
```

not sure what design wants to do if they don't have a primary address, maybe instead of rendering ```null``` we render ```N/A``` or something to that extent?

Reply:
I've been working on enabling TS strict, so that should catch this eventually if it doesn't get resolved. Maybe design has thoughts on a null state. I could see either a sentence about no addresses or removing the whole section as viable options. Is this supposed to display more than one address?

Reply:
I believe it's just the one address